imports/plugins/included/default-theme/client/styles/cards.less

```diff
@@ -83,7 +83,7 @@
 .rui.card .content {
   .flex(1 1 auto);
   padding: 10px;
-  cursor: pointer;
+  cursor: default;
```

Comment:
That the `cursor: default` is not necessary if its not overriding anything.

- **For future consideration** -
And looking forward to Bootstrap 4 cards:  http://v4-alpha.getbootstrap.com/components/card/

I feel like maybe the clickable content might be confusing since there are other action buttons in the card as well. Maybe we always force the use of the arrow buttons.